docs/platform-services/threat-intelligence/about-threat-intelligence.md

```diff
@@ -0,0 +1,65 @@
+---
+slug: /platform-services/threat-intelligence/about-threat-intelligence
+title: About Threat Intelligence
+sidebar_label: About 
+description: Introduction to Sumo Logic's threat intelligence capabilities.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Threat intelligence, often abbreviated as *threat intel*, is information that helps you prevent or mitigate cyber attacks. *Threat intelligence indicators* are individual data points about threats that are gathered from external sources about various entities such as host names, file hashes, IP addresses, and other known targets for compromise. You can import files containing threat intelligence indicators directly into Sumo Logic to aid in security analysis. 
```

Comment:
Maybe you could add this definition to our Glossary?

Reply:
I don't think so. "Threat intelligence indicators" is an industry term, and is not specific to Sumo Logic.